src/main/resources/nl/dtls/fairdatapoint/repository/impl/getFdpIri.sparql

```diff
@@ -0,0 +1,11 @@
+prefix r3d: <http://www.re3data.org/schema/3-0#>
+prefix dcterms: <http://purl.org/dc/terms/>
+
+SELECT DISTINCT ?fdp {
+
+  VALUES ?iri {<INPUT_IRI>}         
```

Comment:
If we bind a value using `TupleQuery.setBinding()` we'll have to see if this is still needed, I don't know yet.